pkg/reconciler/reconciler.go

```diff
@@ -30,12 +30,15 @@ import (
 	"k8s.io/client-go/tools/record"
 	clientset "knative.dev/eventing/pkg/client/clientset/versioned"
 	eventingScheme "knative.dev/eventing/pkg/client/clientset/versioned/scheme"
+	legacyclientset "knative.dev/eventing/pkg/legacyclient/clientset/versioned"
+	legacyScheme "knative.dev/eventing/pkg/legacyclient/clientset/versioned/scheme"
 	"knative.dev/pkg/configmap"
 	"knative.dev/pkg/controller"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/logging/logkey"
 
```

Comment:
Why the separation here? I assume it was to separate `eventing` imports from `pkg` imports but that drifted a little bit?

Reply:
There will be a name collision when I add back a `sources.knative.dev` group.

Reply:
I was only talking about the blank line, just trying to make sense about the way imports are grouped in this file.